src/wcag-link-data.ts

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { DictionaryStringTo } from './dictionary-types';
+
+export interface WCAGLinkData {
+    text: string;
+    url?: string;
+    title?: string;
+}
+
+function standardWcag21Link(
+    urlFormattedTitle: string,
+    sectionNumber: string,
+    title: string,
+): WCAGLinkData {
+    return {
+        text: `WCAG ${sectionNumber}`,
+        title: title,
+        url: `https://www.w3.org/WAI/WCAG21/Understanding/${urlFormattedTitle}`,
+    };
+}
+
+// prettier-ignore
+export const axeTagsToWcagLinkData: DictionaryStringTo<WCAGLinkData> = {
+    wcag111: standardWcag21Link("non-text-content", '1.1.1', 'Non-text Content'),
```

Comment:
Not blocking: I wonder if this would be easier to read/review in the order `sectionNumber`, `title`, `urlFormattedTitle` since then the things that should match (key and `sectionNumber`, `title` and `urlFormattedTitle`) would be adjacent. Actually... on a quick skim it seems like we could programmatically generate `urlFormattedTitle` from `title` (lowercase, remove parens, replace spaces with hyphen), any interest in doing that?

Reply:
I agree on the ordering (fixed now). I tried inferring the urlFormattedTitle and remember finding some edge case where it was ambiguous whether to include a source `-` or not, so I abandoned that.